lib/postinst-migrations.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+#
```

Comment:
Both postinst scripts are setting the `-e` option for the shell script. This is not. Is there a special reason for this (i.e., is this intentional)? Note that in this case, the parent `-e` option will not propagate here AFAIU.

Reply:
That's a good question, @orndorffgrant any reason why we don't have that option here ?

Reply:
This was not intentional - I suppose we should add it for consistency?

Reply:
We could also `source` this script which then would inherit any settings in postinst right?

Reply:
I'll just add `-e`